experimental/dds/tree2/src/feature-libraries/schema-aware/README.md

```diff
@@ -9,8 +9,7 @@ TODO:
         - Normalized should align with `EditableTree`. This will likely require an extra setter on editable fields and on `EditableTree` for extra fields to work around [lacking support for variant accessors for index signatures in TypeScript](https://github.com/microsoft/TypeScript/issues/43826).
     - Allow use in libraries which don't have full schema information:
         - Make unrestricted field types include an extra type in the union that covers the other cases generically.
-    - Support global fields.
-    - Support extra local fields.
+    - Support extra map fields.
     - Support extra global fields (including in a way that is compatible with use in a library that does not have global schema knowledge).
```

Comment:
This line can go too, right?

Reply:
There is still more work to do regarding map nodes/extra fields in the schema aware API, though it does partially work, we want to change how its done, so I think I'll leave this.